addons/mail/static/src/views/activity/activity_model.js

```diff
@@ -13,7 +13,7 @@ export class ActivityModel extends RelationalModel {
 
     async load(params = {}) {
         this.originalDomain = params.domain ? [...params.domain] : [];
-        params.domain = [...params.domain, ["activity_ids", "!=", false]];
+        params.domain = params.domain ? [...params.domain, ["activity_ids", "!=", false]] : [];
```

Comment:
Wouldn't it make sense to keep `["activity_ids", "!=", false]` in the domain? I'm not sure why it shouldn't apply just because no domain is passed in

Reply:
Yup, makes sense. My bad : /
Updated the PR :+1: